Makefile

```diff
@@ -84,3 +91,14 @@ testbin:
 	mkdir -p testbin
 	[[ -x testbin/etcd ]] || curl -L https://storage.googleapis.com/etcd/${ETCD_VER}/etcd-${ETCD_VER}-${OS}-${ARCH}.tar.gz | tar zx -C testbin --strip-components=1 etcd-${ETCD_VER}-${OS}-${ARCH}/etcd
 	[[ -x testbin/kube-apiserver && -x testbin/kubectl ]] || curl -L https://dl.k8s.io/${K8S_VER}/kubernetes-server-${OS}-${ARCH}.tar.gz | tar zx -C testbin --strip-components=3 kubernetes/server/bin/kube-apiserver kubernetes/server/bin/kubectl
+
+# $ make testbin
+#  mkdir -p testbin
+#  [[ -x testbin/etcd ]] || curl -L https://storage.googleapis.com/etcd/v3.4.3/etcd-v3.4.3-darwin-amd64.tar.gz | tar zx -C testbin --strip-components=1 etcd-v3.4.3-darwin-amd64/etcd
+#    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+#                                   Dload  Upload   Total   Spent    Left  Speed
+#  100   205  100   205    0     0    510      0 --:--:-- --:--:-- --:--:--   511
+#  tar: Unrecognized archive format
+#  tar: etcd-v3.4.3-darwin-amd64/etcd: Not found in archive
+#  tar: Error exit delayed from previous errors.
+#  make: *** [Makefile:118: testbin] Error 1
```

Comment:
@joelanford testbin did not work for me. I did not try to fix yet. Just to let you know.

Reply:
- The etcd for mac is:  https://storage.googleapis.com/etcd/${ETCD_VER}/etcd-v3.4.3-darwin-amd64.zip (just the end is wrong)
- However, `kubernetes-server` shows that has not for mac.

Reply:
done a pr: https://github.com/joelanford/helm-operator/pull/18